File: COM_SIC_Activaciones/DataPowerRest/HeadersResponse.vb
'INC000002161718 inicio
Imports System

Public Class HeadersResponse

    Private _HeaderResponse As HeaderResponse

    Public Property HeaderResponse() As HeaderResponse
        Get
            Return _HeaderResponse
        End Get
        Set(ByVal Value As HeaderResponse)
            _HeaderResponse = Value
        End Set
    End Property

    Public Sub New()
        _HeaderResponse = New HeaderResponse
    End Sub
End Class

File: COM_SIC_Procesa_Pagos/DataPowerRest/HeadersResponse.cs
using System;

namespace COM_SIC_Procesa_Pagos.DataPowerRest
{
	[Serializable]
	public class HeadersResponse
	{
		private HeaderResponse _headerResponse;

		public HeaderResponse HeaderResponse
		{
			set{_headerResponse= value;}
			get{ return _headerResponse;}
		}

		public HeadersResponse()
		{
			_headerResponse = new HeaderResponse();
		}
	}
}
